tests-babel/transform.test.js

```diff
@@ -128,11 +129,11 @@ describe('convert templates', () => {
         some content
       */
       {
-        "id": "xn207nfA",
+        "id": "fbPvCNHK",
         "block": "[[[1,\\"some content\\"]],[],false,[]]",
-        "moduleName": "/rewritten-app/a.hbs",
+        "moduleName": "/my-app/components/a/index.gjs",
         "isStrictMode": true
-      }), templateOnly("a.hbs", "a:toc"));"
+      }), templateOnly(undefined, "index:toc"));"
```

Comment:
this doesn't seem right 😅

Reply:
I think this is okay. In the end this will get ignored anyway as moduleName will be used from the template options. This value defaults to '@glimmer/component/template-only' in the constructor of templateOnly. But i can re-verify

Reply:
i confirmed this